hardhat.config.ts

```diff
@@ -97,10 +105,10 @@ const config: HardhatUserConfig = {
   namedAccounts: {
     deployer: {
       default: 0, // take the first account as deployer
-      mainnet: "0x123694886DBf5Ac94DDA07135349534536D14cAf",
+      // mainnet: "0x123694886DBf5Ac94DDA07135349534536D14cAf",
```

Comment:
Should it be `0xfffd7092685bdeebd121d1a0fea3c349114cce50` for the most recent deployment?

Reply:
TBH, I'm not sure what's the point of this value. We had some contracts deployed with `0x123694886DBf5Ac94DDA07135349534536D14cAf` (T, VendingMachines) and some with `0xfffd7092685bdeebd121d1a0fea3c349114cce50` (TokenStaking, proxies, KeepStake). I'd just remove this line.

Reply:
Yeah, or just comment out and leave `""` to make our lives easier for the future (next deployments).

Reply:
Sure, we can do that, but I'm not sure if this is really used, since the script uses the private key from the `CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY` envvar. FWIW, I deployed the contracts with the line commented and I had no problem with that.

Reply:
Ahh, I see. Well, then I'd vote to 🔥 it and maybe just leave a comment explaining it's read from `CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY`.